client/wfnews-war/src/main/angular/src/app/services/notification.service.ts

```diff
@@ -83,6 +87,16 @@ export class NotificationService {
                 return this.http.get(url, { headers }).toPromise()
             })
     }
+
+    public getFireCentreByLocation(bbox): Promise<any> {
```

Comment:
Would be good to define a type or use a type for the bbox

Reply:
added